signinapp/events.py

```diff
@@ -159,23 +186,57 @@ def list_upcoming_events():
     return render_template("events.html.jinja2", prefix="Upcoming ", events=events)
 
 
+@events.route("/events/open")
+def list_open_events():
+    events: list[Event] = db.session.execute(
+        select(Event)
+        .filter_by(enabled=True, registration_open=True)
+        .order_by(Event.start)
```

Comment:
Do we expect to have overlapping time blocks at all? If so, this should `.order_by(Event.start, Event.end)`.

Also if you do `.order_by(Event.start.desc())` then you don't need to reverse the list inside the template